bindings/py/nupic/bindings/algorithms.i

```diff
@@ -1267,6 +1295,62 @@ inline PyObject* generate2DGaussianSample(nupic::UInt32 nrows, nupic::UInt32 nco
     self->getConnectedCounts((nupic::UInt*) PyArray_DATA(x));
   }
 
+  inline PyObject* calculateOverlap_(PyObject* py_inputVector)
+  {
```

Comment:
Why not just make this code part of _calculateOverlap above? Why define a new method?